src/kirby/components/modal/services/modal.helper.ts

```diff
@@ -74,8 +86,56 @@ export class ModalHelper {
     return Promise.resolve(
       baseAnimation
         .addElement(baseEl)
-        .easing('easeOut')
-        .duration(200)
+        .duration(duration)
+        .add(wrapperAnimation)
+        .add(backdropAnimation)
+    );
+  }
+
+  private static animateOut(
+    flavor: any,
```

Comment:
Consider not using "any" and use a type instead

Reply:
fixed